typing/typecore.ml

```diff
@@ -3788,7 +3788,7 @@ and type_cases ?in_function env ty_arg ty_res partial_flag loc caselist =
   in
 (*  if has_gadts then
     Format.printf "lev = %d@.%a@." lev Printtyp.raw_type_expr ty_res; *)
-  (* Do we need to propagate polymorphism *)
+  (* Do we need to propagate polymorphism ? *)
```

Comment:
Interesting... The Joys of spaces and punctuation.

In french :
The comma (, ) and  the point (. ) :
no space before, one space after.

The semicolon ( ; ), the question mark ( ? ), the exclamation mark ( ! ), and the colon ( : ) :
one space before and one space after.